app/controllers/openid_connect/configuration_controller.rb

```diff
@@ -3,12 +3,13 @@ class ConfigurationController < ApplicationController
     def index
       render json: {
```

Comment:
This big response block seems ripe for a refactor out into a service or presenter or view of some kind. Not necessary for this particular PR, but that much code in a controller smells odd to me.